$OpenBSD: patch-configure,v 1.3 2011/01/05 17:47:49 sebastia Exp $

remove parameters that gcc2 doesn't know of, and do not overwrite CFLAGS_OPTIMIZE

--- configure.orig	Thu Jul 15 11:04:48 2010
+++ configure	Wed Dec  8 22:50:09 2010
@@ -6401,12 +6401,12 @@ fi
     ECHO_VERSION='`echo ${PACKAGE_VERSION}`'
     TCL_LIB_VERSIONS_OK=ok
     CFLAGS_DEBUG=-g
-    CFLAGS_OPTIMIZE=-O
+    #CFLAGS_OPTIMIZE=-O
     if test "$GCC" = yes; then :
 
 	# TEA specific:
-	CFLAGS_OPTIMIZE=-O2
-	CFLAGS_WARNING="-W -Wall -Wno-unused-parameter -Wno-missing-field-initializers -ansi -pedantic"
+	CFLAGS_OPTIMIZE=""
+	CFLAGS_WARNING="-W -Wall -ansi -pedantic"
 
 else
   CFLAGS_WARNING=""
@@ -7517,7 +7517,6 @@ fi
 		# OpenBSD builds and links with -pthread, never -lpthread.
 		LIBS=`echo $LIBS | sed s/-lpthread//`
 		CFLAGS="$CFLAGS -pthread"
-		SHLIB_CFLAGS="$SHLIB_CFLAGS -pthread"
 
 fi
 	    # OpenBSD doesn't do version numbers with dots.
